[2025-08-01T05:26:45.046+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-01T05:26:45.146+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_bbbbbbb.extract_data_task manual__2025-08-01T05:26:44.130511+00:00 [queued]>
[2025-08-01T05:26:45.151+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_bbbbbbb.extract_data_task manual__2025-08-01T05:26:44.130511+00:00 [queued]>
[2025-08-01T05:26:45.151+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-01T05:26:45.162+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_data_task> on 2025-08-01 05:26:44.130511+00:00
[2025-08-01T05:26:45.167+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=15125) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-01T05:26:45.168+0000] {standard_task_runner.py:63} INFO - Started process 15127 to run task
[2025-08-01T05:26:45.168+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_bbbbbbb', 'extract_data_task', 'manual__2025-08-01T05:26:44.130511+00:00', '--job-id', '200', '--raw', '--subdir', 'DAGS_FOLDER/dag_generator.py', '--cfg-path', '/tmp/tmpeg4r8_hl']
[2025-08-01T05:26:45.170+0000] {standard_task_runner.py:91} INFO - Job 200: Subtask extract_data_task
[2025-08-01T05:26:45.205+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_bbbbbbb.extract_data_task manual__2025-08-01T05:26:44.130511+00:00 [running]> on host 14614e29d246
[2025-08-01T05:26:45.260+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_bbbbbbb' AIRFLOW_CTX_TASK_ID='extract_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-08-01T05:26:44.130511+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-01T05:26:44.130511+00:00'
[2025-08-01T05:26:45.260+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-01T05:26:45.268+0000] {__init__.py:57} INFO - 執行 extract_data (資料載入層)
[2025-08-01T05:26:45.268+0000] {__init__.py:61} INFO - 呼叫演算法 'test_csv' 來自模組 'processor.extract.readCSV'
[2025-08-01T05:26:45.271+0000] {__init__.py:71} INFO - 資料是 pd.DataFrame
[2025-08-01T05:26:45.281+0000] {__init__.py:82} INFO - 資料已存入 Redis，key=***_data:dag_bbbbbbb.extract_data_task
[2025-08-01T05:26:45.292+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-01T05:26:45.292+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-01T05:26:45.299+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_bbbbbbb, task_id=extract_data_task, run_id=manual__2025-08-01T05:26:44.130511+00:00, execution_date=20250801T052644, start_date=20250801T052645, end_date=20250801T052645
[2025-08-01T05:26:45.341+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-01T05:26:45.357+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-01T05:26:45.360+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
